content/yaml/building-a-native-android-app.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Building a native Android app
+description: Building an Android app with YAML.
+weight: 3
+---
+## Building an Android app
+
+The necessary command for building native Android application goes under `scripts` in the [overall architecture](../yaml/yaml/#template) in the `codemagic.yaml` file. For Android (built with gradle), the script looks like this:
+
+    - ./gradlew bundleDebug
+
+## Testing an Android app
+
+The code for testing an Android app also goes under `scripts`. A few examples of testing can be found [here](../yaml/testing).
+
+## Code signing
+
+For gradle code signing configuration refer to the [documentation](https://docs.codemagic.io/code-signing/android-code-signing/#preparing-your-flutter-project-for-code-signing). More information about code singing with YAML in general is [here](../yaml/distribution).
+
+## Publishing
```

Comment:
would be nice to mention that these are the available integrations but it's possible also to publish elsewhere with custom scripts (e.g. firebase app distribution)